src/epsilonServer.cpp

```diff
@@ -15,8 +16,12 @@ EpsilonServer::EpsilonServer()
     for(unsigned int n=0; n<factionInfo.size(); n++)
         factionInfo[n]->reset();
 
-    state_logger = new GameStateLogger();
-    state_logger->start();
+    if (PreferencesManager::get("game_logs", "1").toInt())
+    {
+        state_logger = new GameStateLogger();
+        state_logger->start();
+    } else if (state_logger)
+        state_logger->destroy();
```

Comment:
Yeah, that makes sense. I've removed it.

Should the game state logger start in gameGlobalInfo->startScenario() instead of server creation? If logging starts there, it could be toggled from the server creation screen. It doesn't look like anything's logged until the scenario's started anyway.